source/apiVolontaria/volunteer/serializers.py

```diff
@@ -414,11 +415,39 @@ class ParticipationBasicSerializer(serializers.ModelSerializer):
 
     # Explicitly declare the BooleanField to make it "required"
     standby = serializers.BooleanField()
+    invit_icalandar = serializers.BooleanField(
+        default=False,
+        write_only=True,
+    )
     user = UserBasicSerializer(
         read_only=True,
         default=serializers.CurrentUserDefault(),
     )
 
+    def create(self, validated_data):
+        participation = models.Participation()
+        participation.standby = validated_data['standby']
+        participation.event = validated_data['event']
+        participation.user = validated_data['user']
+
+        if 'invit_icalandar' in validated_data.keys():
+            participation.presence_duration_minutes = validated_data['invit_icalandar']
+            name = "Volontaria event - " + participation.event.cycle.name
```

Comment:
I18N function need to be used here

Reply:
@RignonNoel je ne comprends pas pourquoi tu voudrais une traductions pour le nom de l'event

Reply:
done